content/setting-up-tugboat/connect-with-your-provider.md

```diff
@@ -274,3 +275,103 @@ You'll see a dialog box explaining what happens if you disconnect a linked accou
 ![Dialog box explaining disconnecting a linked git provider account with arrow to "Disconnect" button](../../_images/press-disconnect-button.png)
 
 {{% /expand%}}
+
+## Authenticating using an API access token
+
+By default, Tugboat uses OAuth to authenticate to git hosting providers. However, there may be times when GitHub,
+GitLab, or Bitbucket organizations take too long to approve org-level OAuth applications. As a workaround, you can use a
+git provider's API access token to add a GitHub, GitLab, or Bitbucket repo to Tugboat.
+
+You'll need to use the [Tugboat API](https://api.tugboat.qa/) to pass an alternate authentication method.
+
+This example shows how to implement this workaround for a GitHub repository, but you can do the same thing using a
+slightly different syntax for GitLab or Bitbucket repositories. If you need help with the syntax for GitLab or
+Bitbucket, or have questions about this workaround, reach out to us for [support](/support).
```

Comment:
Can we link to the API documentation here where we describe this endpoint, in addition to the support link?

https://api.tugboat.qa/v3#tag/Repositories/paths/~1repos/post